.github/actions/package/action.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Package
+description: Install python and package
+inputs:
+  python-version:
+    description: "Python version"
+    required: true
+
+runs:
+    using: "composite"
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - uses: actions/cache@v3
```

Comment:
What is scope of this cache?
If more than one action:
* when the cache invalidates?
* we would like to install dependencies on e2e daily workflow.

If only one action but common for all workflows, then ok.

Reply:
To my knowledge it's global and I've made the current date a part of the cache name so it invalidates on daily basis. Check the list of currently cached deps here: https://github.com/neptune-ai/neptune-client/actions/caches